src/Runner/Runner.php

```diff
@@ -300,35 +301,21 @@ function (array $workerResponse) use ($processPool, $process, $identifier, $getF
                     }
 
                     if (ParallelAction::WORKER_ERROR_REPORT === $workerResponse['action']) {
-                        $this->errorsManager->reportWorkerError(new WorkerError(
-                            $workerResponse['message'],
-                            $workerResponse['file'],
-                            (int) $workerResponse['line'],
-                            (int) $workerResponse['code'],
-                            $workerResponse['trace']
-                        ));
-
-                        return;
+                        throw SourceExceptionFactory::fromArray($workerResponse); // @phpstan-ignore-line
```

Comment:
worker crashed here, shouldn't we use `WorkerException::fromRaw` here ?

Reply:
I though we could try represent it with original error, but yeah, it makes more sense to use `WorkerException::frowRaw()` here. I've changed it during interactive rebase.